File: src/test/java/com/ice/PersonTest.java
package com.ice;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class PersonTest {

    @Test
    public void bmiTest() {
        Person person = new Person(63.3f,1.8f);
        Assertions.assertEquals(63.3f/(1.8f*1.8f),person.bmi());
    }
}

File: src/test/kotlin/com/ice/PersonTest.kt
package com.ice.kotlin

import com.ice.Human
import com.ice.Person
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class PersonTest {
    @Test
    fun bmiTest(){
        val human = Human("ice", 63.3f, 1.8f)
        Assertions.assertEquals(63.3f/(1.8f*1.8f), human.bmi())
    }
    @Test
    fun personBmiTest(){
        val personTest = Person(63.3f, 1.8f)
        Assertions.assertEquals(63.3f/(1.8f*1.8f), personTest.bmi())
    }
}